setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+
+setup(
```

Comment:
You mentioned adding "pyproject.toml" but it seems you actually added setup.py ?

Reply:
The initial PR did use a `pyproject.toml` but that ended up adding a dependency (I thought `flit_core` was built in to `pip` but it seems that we can't rely on that, perhaps due to system packaged versions of `pip`), so we moved to the more consistently supported `setup.py`.